Tests/test_imagefont.py

```diff
@@ -28,497 +28,527 @@
 pytestmark = skip_unless_feature("freetype2")
 
 
-class TestImageFont:
-    LAYOUT_ENGINE = ImageFont.Layout.BASIC
+def test_sanity():
+    assert re.search(r"\d+\.\d+\.\d+$", features.version_module("freetype2"))
+
+
+@pytest.fixture(
+    scope="module",
+    params=[
+        ImageFont.Layout.BASIC,
+        pytest.param(ImageFont.Layout.RAQM, marks=skip_unless_feature("raqm")),
```

Comment:
Can we do this for consistency?
```suggestion
        pytest.param(ImageFont.Layout.BASIC),
        pytest.param(ImageFont.Layout.RAQM, marks=skip_unless_feature("raqm")),
```

Reply:
Good idea, seems to work fine.